README.md

```diff
@@ -344,6 +344,87 @@ $ cargo install scx_rusty
 
 and `scx_rusty` will be built and installed as `~/.cargo/bin/scx_rusty`.
 
+## Checking scx_stats
+
+- To check the scheduler statistics, use the
+
+```
+scx_SCHEDNAME --monitor SEC
+```
+
+for example 0.5 - this will print the output every half a second
+
+```
+scx_bpfland --monitor 0.5
+```
+
+#### Examples
+
+- scx_bpfland
+
+```
+❯ scx_bpfland --monitor 5
+[scx_bpfland] tasks -> run:  3/4  int: 2  wait: 3    | nvcsw: 3    | dispatch -> dir: 0     prio: 73    shr: 9
+[scx_bpfland] tasks -> run:  4/4  int: 2  wait: 2    | nvcsw: 3    | dispatch -> dir: 1     prio: 3498  shr: 1385
+[scx_bpfland] tasks -> run:  4/4  int: 2  wait: 2    | nvcsw: 3    | dispatch -> dir: 1     prio: 2492  shr: 1311
+[scx_bpfland] tasks -> run:  4/4  int: 2  wait: 3    | nvcsw: 3    | dispatch -> dir: 2     prio: 3270  shr: 1748
+```
+
+- scx_rusty
+
+```
+❯ scx_rusty --monitor 5
+###### Thu, 29 Aug 2024 14:42:37 +0200, load balance @  -265.1ms ######
+cpu=   0.00 load=    0.17 mig=0 task_err=0 lb_data_err=0 time_used= 0.0ms
+tot=     15 sync_prev_idle= 0.00 wsync= 0.00
+prev_idle= 0.00 greedy_idle= 0.00 pin= 0.00
+dir= 0.00 dir_greedy= 0.00 dir_greedy_far= 0.00
+dsq=100.00 greedy_local= 0.00 greedy_xnuma= 0.00
+kick_greedy= 0.00 rep= 0.00
+dl_clamp=33.33 dl_preset=93.33
+slice=20000us
+direct_greedy_cpus=f
+  kick_greedy_cpus=f
+  NODE[00] load=  0.17 imbal=  +0.00 delta=  +0.00
+   DOM[00] load=  0.17 imbal=  +0.00 delta=  +0.00
+###### Thu, 29 Aug 2024 14:42:42 +0200, load balance @ -1268.9ms ######
```

Comment:
Would also reduce here the logs, so that the README is not too much filled with logs.

Reply:
Fixed